go.mod

```diff
@@ -43,7 +43,7 @@ require (
 	github.com/pingcap/goleveldb v0.0.0-20171020122428-b9ff6c35079e
 	github.com/pingcap/kvproto v0.0.0-20190619024611-a4759dfe3753
 	github.com/pingcap/log v0.0.0-20190307075452-bd41d9273596
-	github.com/pingcap/parser v0.0.0-20190613082312-d2cf6071823d
+	github.com/pingcap/parser v0.0.0-20190627064259-e9c42442aa72
```

Comment:
Why need to change parser?

Reply:
To include this fix https://github.com/pingcap/parser/pull/366
The SQL hint is not successful parsed in `select /*+ MAX_EXECUTION_TIME(1000) */ SLEEP(5);`
@crazycs520